src/scripts/main.js

```diff
@@ -21,7 +21,34 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key1 in food) {
+    const ol = document.createElement('ol');
+
+    ol.textContent = key1;
+    document.body.appendChild(ol);
+
+    if (typeof food[key1] === 'object') {
+      for (const key2 in food[key1]) {
+        if (key2 !== undefined) {
```

Comment:
The check `if (key2 !== undefined)` is unnecessary because `for...in` loops only iterate over defined keys.